lib/node_modules/@stdlib/blas/base/drot/docs/types/index.d.ts

```diff
@@ -0,0 +1,113 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/**
+* Interface describing `drot`.
+*/
+interface Routine {
+	/**
+	* Applies plane rotation.
+	*
+	* @param N - number of indexed elements
+	* @param x - first input array
+	* @param strideX - `x` stride length
+	* @param y - second input array
+	* @param strideY - `y` stride length
+	* @param c - double precision floating-point number
+	* @param s - double precision floating-point number
+	* @returns `y`
+	*
+	* @example
+	* var Float64Array = require( '@stdlib/array/float64' );
+	*
+	* var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+	* var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+	*
+	* drot( x.length, x, 1, y, 1, 0.707, 0.707 );
+	* // x => <Float64Array>[ 4.949, 6.3629999999999995, 7.776999999999999, 9.190999999999999, 10.604999999999999 ]
+	* // y => <Float64Array>[ 3.535, 3.535, 3.5349999999999997, 3.5349999999999997, 3.5349999999999997 ]
+	*/
+	( N: number, x: Float64Array, strideX: number, y: Float64Array, strideY: number, c: number, s: number ): Float64Array;
+
+	/**
+	* Applies plane rotation.
+	*
+	* @param N - number of indexed elements
+	* @param x - first input array
+	* @param strideX - `x` stride length
+	* @param offsetX - starting index for `x`
+	* @param y - second input array
+	* @param strideY - `y` stride length
+	* @param offsetY - starting index for `y`
+	* @param c - double precision floating-point number
+	* @param s - double precision floating-point number
+	* @returns `y`
+	*
+	* @example
+	* var Float64Array = require( '@stdlib/array/float64' );
+	*
+	* var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+	* var y = new Float64Array( [ 7.0, 8.0, 9.0, 10.0, 11.0, 12.0 ] );
+	*
+	* drot.ndarray( 4, x, 0, 2, y, 0, 2, 0.707, 0.707 );
+	* // x => <Float64Array>[ 1.0, 2.0, -2.998188273612, 4.0, 5.0, 6.0 ]
+	* // y => <Float64Array>[ 7.0, 8.0, -8.994564820835999, 10.0, 11.0, 12.0 ]
```

Comment:
Same comment as above. And also, you need to have the example use non-zero strides.